### ISSUE TITLE
binutils version 2.33 renames bfd_get_section_flag macro

### ISSUE THREAD

**Issue by lhw** [gnif/LookingGlass#232]
With commit [fd361982](https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;h=fd3619828e94a24a92cddec42cbc0ab33352eeb4) of binutils some of the macros were renamed, including bfd_get_section_flag. This was reported on the [debian bugtracker](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=948529). My simple fix was to switch the macro for now but this would of course break compatibility with older versions of binutils. Concerning the other two points he brought up thats more of a debian workaround.

**Comment by chrsm** [comment]
This got it to build and run for me, though I'm not sure if it's 100% correct :-)

```c
diff --git a/common/src/platform/linux/crash.c b/common/src/platform/linux/crash.c
index 218d65f..7b08a41 100644
--- a/common/src/platform/linux/crash.c
+++ b/common/src/platform/linux/crash.c
@@ -98,10 +98,10 @@ static void load_symbols()

 static bool lookup_address(bfd_vma pc, const char ** filename, const char ** function, unsigned int * line, unsigned int * discriminator)
 {
-  if ((bfd_get_section_flags(crash.fd, crash.section) & SEC_ALLOC) == 0)
+  if ((bfd_section_flags(crash.section) & SEC_ALLOC) == 0)
     return false;

-  bfd_size_type size = bfd_get_section_size(crash.section);
+  bfd_size_type size = bfd_section_size(crash.section);
   if (pc >= size)
     return false;

@@ -230,4 +230,4 @@ bool installCrashHandler(const char * exe)
   return true;
 }

-#endif
\ No newline at end of file
+#endif
```

**Comment by gnif** [comment]
The change is simple but for now, it's on hold as only those running on unstable/bleeding edge versions of Binutils are affected. Really this should have some `ifdefs` to select the correct code depending on the binutils version.

**Comment by chrsm** [comment]
@gnif Sorry for the delay. I made a PR with these changes properly ifdef'd.

https://github.com/gnif/LookingGlass/pull/241

**Comment by gnif** [comment]
Looks and tests good, thanks!

Fixed in #241
